website/docs/docs/build/python-models.md

```diff
@@ -16,26 +16,20 @@ We encourage you to:
 - Weigh in on [next steps for Python models, beyond v1.3](https://github.com/dbt-labs/dbt-core/discussions/5742).
```

Comment:
```suggestion
- Share your thoughts and ideas on [next steps for Python models](https://github.com/dbt-labs/dbt-core/discussions/5742).
```

Reply:
sugg:

- "weigh in" could be colloquial so more challenging to translate 
- remove "1.3" for more future proofing / easier docs maintenance